xml/System.ServiceModel.Syndication/SyndicationFeed.xml

```diff
@@ -1982,9 +2040,38 @@
         <ReturnType>System.ServiceModel.Syndication.SyndicationTextInput</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets or sets the TextInput property for the feed.</summary>
+        <value>A <see cref="T:System.ServiceModel.Syndication.SyndicationTextInput" /> instance that contains required parameters for the RSS 'textInput' item for the feed.</value>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ The RSS specification allows for a 'textInput' item to be included within a syndication feed that the publisher can userinput
+ to direct search requests or obtain feedback through a simple text input box and button. A 'textInput' element in an RSS feed
```

Comment:
Is this redundant? The documentation for SyndicationTextInput already contains this information.

Reply:
Maybe these remarks could be substituted with a comment suggesting to look at the remarks in `SindicationTextInput` for more information (or viceversa).